pkg/aws/awsmachinedeployment/validate_awsmachinedeployment.go

```diff
@@ -119,6 +136,23 @@ func (v *Validator) ValidateCreate(request *admissionv1.AdmissionRequest) (bool,
 	return true, nil
 }
 
+func (v *Validator) InstanceTypeValid(awsMachineDeployment infrastructurev1alpha2.AWSMachineDeployment) error {
+	if !contains(v.validInstanceTypes, awsMachineDeployment.Spec.Provider.Worker.InstanceType) {
+		v.logger.Log("level", "debug", "message", fmt.Sprintf("AWSMachineDeployment %s worker instance type %v is invalid. Valid instance types are: %v",
```

Comment:
We already logging the error here https://github.com/giantswarm/aws-admission-controller/blob/master/pkg/validator/handler.go#L64, no need to for that.

Reply:
You are right! I meant to drop this and just realized I did it in the `awscontrolplane` and not here.